modules/nlu/src/views/entities/index.jsx

```diff
@@ -0,0 +1,124 @@
+import React from 'react'
+import { Button, FormGroup, FormControl, InputGroup, Glyphicon, ListGroup, ListGroupItem, Label } from 'react-bootstrap'
+
+import style from '../style.scss'
+import EntityEditor from './editor'
+
+export default class EntitiesComponent extends React.Component {
+  state = {
+    entities: [],
+    filteredEntities: [],
+    selectedEntity: undefined
+  }
+
+  componentDidMount() {
+    this.fetchEntities()
+  }
+
+  fetchEntities = () => {
+    this.props.bp.axios.get('/mod/nlu/entities').then(res => {
+      const customEntities = res.data.filter(r => r.type !== 'system')
+      this.setState({
+        entities: customEntities,
+        filteredEntities: customEntities,
+        selectedEntity: customEntities[0]
```

Comment:
this could be null if no entity is returned

Reply:
thats the intended behaviour. If no entities are returned, selectedEntity will be undefined like in its initial state.